### PR TITLE
Feedback

### PR DESCRIPTION
**Pull request by jia200x**
This PR adds feedback to frontend and reception in backend. Feedback can be read from the django admin.

### CODE REVIEW

**Comment by jia200x** [comment]
tested last changes, work ok for me.

File: django/api/models.py [hunk 1]
```diff
@@ -119,3 +119,8 @@ class ApplicationInstance(models.Model):
     class Meta:
         permissions = (('has_dev_perm','Has dev permissions'),)
         unique_together = ('version_code', 'application',)
+
+class Feedback(models.Model):
+    date = models.DateTimeField(auto_now=True)
+    description = models.TextField(blank=True)
```
Comment (HendrikVE) [inline django/api/models.py]: We should only allow non-empty feedback

Comment (HendrikVE) [reply django/api/models.py]: But its working for me and its just a tweak so..

ACK

Comment (jia200x) [reply django/api/models.py]: I didn't want to add this logic in the frontend since it's only a tool for us developers

**Comment by jia200x** [comment]
Go

**Pull request merged**